validate/validate.go

```diff
@@ -519,6 +520,12 @@ func (v *Validator) CheckLinux() (msgs []string) {
 		}
 	}
 
+	if v.spec.Linux.MountLabel != "" {
+		if err := label.Validate(v.spec.Linux.MountLabel); err != nil {
```

Comment:
I'm not sure what `label.Validate` is checking, but the spec [places no limits on the `linux.mountLabel` value][1].  If we want to validate it (against a particular kernel API?), then I think we probably need clarity in the spec about what API the value will be passed to, and the check here should probably only happen when `HostSpecific` is set.

[1]: https://github.com/opencontainers/runtime-spec/blob/fae0d09a24b78ef17e43d411583780b9f2a48d5b/config-linux.md#mount-label

Reply:
I still don't see spec grounds for this check, but if the runtime-tools maintainers feel like it's grounded, then the `validate/` changes from this PR belong in [v0.3.0][1].  That could either happen by spinning them out into a new PR, or by reviewing them here in the context of the runtimtetest changes.

[1]: https://github.com/opencontainers/runtime-tools/milestone/5